lib/grammers-tl-gen/src/lib.rs

```diff
@@ -44,10 +47,33 @@ const SPECIAL_CASED_TYPES: [&str; 1] = ["Bool"];
 fn ignore_type(ty: &Type) -> bool {
     SPECIAL_CASED_TYPES.iter().any(|&x| x == ty.name)
 }
+#[derive(Debug, PartialEq)]
+pub struct GeneratableDefinition {
+    pub parsed: Definition,
+    pub docs: Option<DocumentationItem>,
+}
+#[derive(Debug, Clone, PartialEq, serde::Deserialize)]
+pub struct TlError {
```

Comment:
`RpcError` might be a more accurate name.

Reply:
I copied it from scrape-docs code

Reply:
Right, a mistake on my part.